pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -30,10 +30,14 @@ import (
 
 const (
 	rootDir              = "/opt/tectonic"
-	defaultReleaseImage  = "registry.svc.ci.openshift.org/openshift/origin-release:v4.0"
+	defaultReleaseImage  = "registry.svc.ci.openshift.org/openshift/origin-release:latest"
```

Comment:
I don't think we want this.  The `v4.0` tag is already a mutable, floating tag.  For pinning, we'll probably want to resolve it into a digest, since those are inherently immutable.  But we can build the "look up the digest" code into a separate helper.  For this PR, I expect we should leave this line as it was with `v4.0`.